src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -372,7 +372,171 @@ private XmlDocCommentTextFormatDefinition()
         }
         #endregion
 
-        #region VB XML Literals - Attribute Name 
+        #region Regex - Comment
+
+#if dark_theme
+        private static readonly Color s_regexTextColor = Color.FromRgb(0xd6, 0x9d, 0x85);
+        private static readonly Color s_regexOtherEscapeColor = Color.FromRgb(0xff, 0xd6, 0x8f);
+        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x05, 0xc3, 0xba);
+        private static readonly Color s_characterClassColor = Color.FromRgb(0x00, 0x8a, 0xff);
+        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0xd7, 0x45, 0x8c);
+        private static readonly Color s_regexCommentColor = Color.FromRgb(87, 166, 74);
+#else
+        private static readonly Color s_regexTextColor = Color.FromRgb(0x80, 0x00, 0x00);
+        private static readonly Color s_regexOtherEscapeColor = Color.FromRgb(0x9e, 0x5b, 0x71);
+        private static readonly Color s_regexGroupingAndAlternationColor = Color.FromRgb(0x05, 0xc3, 0xba);
+        private static readonly Color s_characterClassColor = Color.FromRgb(0x00, 0x73, 0xff);
+        private static readonly Color s_regexAnchorAndQuantifierColor = Color.FromRgb(0xff, 0x00, 0xc1);
+        private static readonly Color s_regexCommentColor = Color.FromRgb(0, 128, 0);
```

Comment:
when this goes in, we'll need to open a tracking bug to get the dark theme colors checked in.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/29298](https://github.com/dotnet/roslyn/issues/29298) for tracking

---
In reply to: [210114960](https://github.com/dotnet/roslyn/pull/25517#discussion_r210114960) [](ancestors = 210114960)